book.py

```diff
@@ -0,0 +1,107 @@
+#Program to make a simple calculator
+import math
+print("\__CALCULATOR__")
+
+def sum(a,b):
+    a+=b
+    return a
+def sub(a,b):
+    if a>b:
+        a-=b
+        return a
+    else:
+        b-=a
+        return b
+
+def mul(a,b):
+    a*=b
+    return a
+
+def div(a,b):
+    q=a/b
+    r=a%b
+    print("\nThe quotient  is : %s"%q)
+    print("\nThe remainder  is :%s"%r)
+
+def sqr(a):
+    x=math.sqrt(a)
+    return x
+
+while(True):
+    print("\n\nChoose the operation you want to perform: ")
+    print("\n\t1.ADDITION")
+    print("\n\t2.SUBTRACTION")
+    print("\n\t3.MULTIPLICATION")
+    print("\n\t4.DIVISION")
+    print("\n\t5.SQUARE ROOT")
+    print("\n\t6.EXIT")
+
+    choice =int(input('>'))
+    if choice==1:
+        print("\n\nEnter the two numbers: ")
+        num1 = int(input('>'))
+        num2 = int(input('>'))
+        s=sum(num1,num2)
+        print("The sun is : %s" %s)
```

Comment:
Typo here
>print("The sun is  :  ...

It should be:
>print("The sum is :  ...

Reply:
but sir its just input error . maybe quoted section couldn't affect whole code